### DISCUSSION TITLE
Sugerencias de Logo para el equipo :dagger:  :imp:  :smiling_imp: :imp:  :dagger:

### DISCUSSION THREAD

**Discussion by BraianTroncoso** [CodeSystem2022/Debugging_Demons_Tercer_Semestre#24] [Ideas]
Dejen imagenes o referencias que quieran como logo.

**Comment by BraianTroncoso** [comment]
Acá dejo 2 realizados por mi: :smiling_imp: 
![logoRojo](https://user-images.githubusercontent.com/95662710/236388152-c02ec105-bc3c-403d-8bd0-ebb1c98d134a.png)
![logoMoradoConLLaveDeCerrado](https://user-images.githubusercontent.com/95662710/236388156-efb1226c-9a02-4e44-b314-baf23af0cbc3.png)

**Comment by francogenre** [comment]
A mí no me parecía mal este que habían subido al grupo de WhatsApp y que después usé para la plantilla que usamos para los prácticos, pero estos están buenos también. No tiene fondo, por eso se ve así.
![Demon](https://user-images.githubusercontent.com/84336216/236957317-21eb33fb-5b16-4bd8-b15e-e1fa22068050.png)

**Reply by BraianTroncoso** [reply]
Esta genial!!

**Comment by Aguppesce** [comment]
Acá hice uno desde esta [página](https://www.designevo.com/apps/logo/), hay varios más, se puede modificar a gusto, lo deje así, pero esta para colgarse un rato 

![imagen](https://user-images.githubusercontent.com/56004874/236960267-b5b07690-eee5-4ca9-a3c2-fe582203ecbb.png)

**Reply by BraianTroncoso** [reply]
Copado bro!!
